src/DatatypeChannels.ASB/DatatypeChannels.ASB.fsproj

```diff
@@ -27,8 +27,7 @@
     <Compile Include="DatatypeChannels.fs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="6.0.6" />
```

Comment:
Is this not needed? What version ends up being referenced?

Reply:
It uses current SDK's version unless the project file overrides it. There is an issue with the latter, but the former works as expected.